recipes/toolchain/bld.bat

```diff
@@ -0,0 +1,6 @@
+setlocal EnableDelayedExpansion
+
+FOR %%F IN (activate deactivate) DO (
```

Comment:
Sure. Just for edification, this is doing the same thing that `build.sh` does except for Windows, which is copy the `activate.bat` and `deactivate.bat` into the proper directories. This allows them to run when the environment is activated.